src/visualizr/anitalker/dataset.py

```
@@ -282,0 +1,298 @@
+
+                    item_dict["frame_count"] = min_len
+                    item_dict["hubert_obj"] = item_dict["hubert_obj"][
+                        :,
+                        : min_len * 2,
+                        :,
+                    ]
+
+                if min_len < self.window_size * self.video_fps + 5:
+                    continue
+
+        print("Db count:", len(self.data))
```

Ruff (0.12.2)

`print` found

Remove `print`

(T201)